docs/tools/offline/cli-device-management.md

```diff
@@ -0,0 +1,76 @@
+### Updating Devices with Mbed CLI
+
+Mbed CLI includes features that help prepare and ship updates for devices managed through Pelion. The Mbed CLI uses the subcommands starting with `mbed device-management`, `mbed dev-mgmt` or `mbed dm` to manage devices.
+
+Start by configuring your Mbed Cloud SDK API Key, target and toolchain. Obtain the API Key from the Pelion portal.
+
+```
+$ mbed config -G CLOUD_SDK_API_KEY ak_...
+$ mbed target K64F
+$ mbed toolchain GCC_ARM
+```
+
+Next, initialize the device management feature of Mbed CLI with the following command:
+
+```
+$ mbed device-management init -d "<company domain name>" -m "<product model identifier>"
+```
+
+First, this command asks for information about your update certificate. When Mbed CLI has enough information, it creates several files:
+* A certificate in `.update-certificates/default.der`.
+* A matching private key in `.update-certificates/default.key.pem`.
+* A set of default settings in `.manifest_tool.json`.
+* Mbed Cloud settings in `.mbed_cloud_config.json`.
+* Mbed Cloud developer credentials in `mbed_cloud_dev_credentials.c`
+* Mbed Cloud update credentials in `update_defalut_resources.c`
+
+The default settings include:
+* A unique vendor identifier, based on the domain name supplied to `init`.
+* A unique model identifier, based on the vendor identifier and the model name supplied to `init`.
+* The path of the certificate and private key.
+
+If you do not want to enter the subject information for your update certificate (country, state, city, organization and so on), add the `-q` flag to the command above.
+
+As `device-management` is very long to type, the remainder of this document will use the `mbed dm` alias for all device management subcommands.
+
+<span class="notes">**Note:** The certificate created in `mbed dm init` is not suitable for production. Use it for testing and development only. To create a certificate for production purposes, please use an air-gapped computer or a Hardware Security Module. When going to production, conduct a security review on your manifest signing infrastructure, as it is the core of the security guarantees for update client.</span>
+
+#### Single-device update
+Once you have run `mbed device-management init`, you can perform updates on a single device by:
```

Comment:
Device Management isn't shortened here?

Reply:
We also haven't been 'allowed' to shorten it on our side, but I'm not sure that requirements still stands for what things are named in code. Also not willing to risk making that decision myself  😑

Reply:
Code is its own thing; they need to sort it out between them

Reply:
I don't like using "once you have x". Also, why is single-device hyphenated?

Reply:
@iriark01 I'll reword it. Single device is not hyphenated as that's not how I would write. I'll drop single.